### ISSUE TITLE
Definition of Q in unstable Paux<0 regions revisited?

### ISSUE THREAD

**Issue by pabloprf** [cfs-energy/cfspopcon#25]
After an intense _whats-going-on!_ session with Audrey today we realized there might be better ways to define Q in the region of Paux<0.
In this line, Q is defined as Pfusion/Plaunched
https://github.com/cfs-energy/cfspopcon/blob/895fb097e4d2716c2a55abd49676cacf55cd91a0/cfspopcon/formulas/Q_thermal_gain_factor.py#L39

Assuming perfect absorption, Pexternal=Plaunched.
Pexternal comes from Pin-Palpha, and with the separated calculation of Pohmic, this may lead to situations of Paux<0 (if Pohmic is higher than Pexternal). The meaning of this is that the point is not in power balance, dW/dt (ignored in POPCONs) would be positive, then raising the temperature at constant density until a stable point (Paux=0) is found.

On the other hand, the calculation of Q will use a Pexternal which is lower than the "naturally ocurring" Pohmic, thus biasing the Q contours and having them artificially high in that region.
While this is solved by using a constraint of the type Paux>0 when defining an operational window (as in the PRD template), if one doesn't use this constraint, an optimizer that tries to find the maximum Q will drive the system towards that region always.

A suggestion to improve the definition of Q is to define it as Pfusion/max(Pohmic, Plaunched), instead of Pfusion/Plaunched.
There might be better ways to define this, but this may work.

Thoughts @tbody-cfs @hassec ?

**Comment by djbattagl** [comment]
@pabloprf I think the proposed solution is fine. Another option is to define Q to be zero when Paux < 0.  Another option is to have optimization algorithm penalize solutions when Paux < 0.

**Comment by pabloprf** [comment]
Yes, that's also fine. I guess it's a decision on how dW/dt is accounted for in the fusion gain calculation. However, even if dW/dt is substracted in the denominator of Q, my concern is that POPCONs should be dealing with stationary solutions, and the points with Paux<0 aren't.
